doc/glossary.rst

```diff
@@ -0,0 +1,165 @@
+
+Digital Signal Processing
+-------------------------
+
+For a general introduction to digital signal processing, we recommend
+`Seeing Circles Sines and Signal <https://jackschaedler.github.io/circles-sines-signals/>`_
+by Jack Schaedler.
+
+.. glossary::
+
+    time domain
+        Signals that are represented as variations over time, and analyses of such signals.
+    frequency domain
+        Signals that are represented in terms of frequencies, and Analyses of such signals.
+    temporal resolution
+        The precision of a measurement, in the time domain.
+    frequency resolution
+        The precision of a measurement, in the frequency domain.
+    sampling rate
+        The rate at which samples are taken.
+
+Units
+-----
+
+.. glossary::
+
+    Hertz (Hz)
+        A unit of frequency, as the number of cycles per second.
+    Decibels (dB)
+        A unit of intensity, on a logarithmic scale.
+    Volts (V)
+        A unit of voltage, typically in the microvolt (uV) range for neural time series.
+
+Filters
+-------
+
+For a guide on filtering, specific to electrophysiological data, check out this
+`paper <https://doi.org/10.1016/j.jneumeth.2014.08.002>`_ from the journal of neuroscience methods.
+
+For a more in depth tutorial, in code, check out the
+`MNE Filtering Tutorial <https://martinos.org/mne/stable/auto_tutorials/plot_background_filtering.html>`_.
+
+.. glossary::
+
+    Impulse Response
+        The response of a fitler when presented with an impulse; a single, brief input.
+    FIR
+        A Finite Impulse Response filter, meaning the response to a single input is finite, settling to zero.
+    IIR
+        An Infinite Impulse Response filter, meaning the response to a single input continue infinitely.
+    passband
+        The range (band) of frequencies that can pass through a filter.
+    stopband
+        The range (band) of frequencies that are attenuated (stopped) by a filter.
+    passtype
+        The type of filter, defined in terms of what frequency bands or ranges it passes through, or filters out.
+
+        * bandpass: a filter whose passband is a specific frequency band, bound by a low and high frequency point.
+        * bandstop: a filter that passes through all frequencies except a band region that is attenuated.
+        * lowpass: a filter whose passband is all frequencies below a filter frequency (low frequencies pass through).
+        * highpass: a filter whose passband is all frequencies above a filter frequency (high frequencies pass through).
+    transition band
+        The range of frequencies that are in the transition region between the passband and the stopband.
+    frequency response
+        The response profile of a filter, specifying the gain and phase shift applied by the filter at each frequency.
+
+Rhythms & Bursts
+----------------
+
+.. glossary::
+
+    burst
+        Periodic activity that lasts for a finite time , as in a 'burst of oscillatory activity'.
```

Comment:
I would probably say a "short" time, instead of finite time. Because all oscillations end in finite time, esp if you die.